recipes/wntr/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "wntr" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/michaelbynum/WNTR/archive/f73f0618e22201c0d15e5d1c62537564fc8840cb.zip
+  sha256: ed3b323ebe7bac1cce88520294327f61327d752f0e5086a28c909896948f799b
+
+build:
+  number: 0
+  skip: true  # [win]
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - numpy
+    - python
+    - pip
+  host:
+    - python
+    - pip
+    - numpy
+  run:
+    - python
+    - {{ pin_compatible('numpy') }}
```

Comment:
If you're not compiling code based on the numpy c-api, you don't need the `pin_compatible` entry.